lib/formulas/qunit-transforms.js

```diff
@@ -98,12 +116,15 @@ module.exports = function transform(source) {
   var sections = {
     addQUnitImport: true,
     modules: [],
+    moduleCallbacks: [],
     tests: [],
     skips: [],
     assertions: []
   };
 
   var ast = recast.parse(source);
+  var isInSetup = false;
```

Comment:
I'm  in general :-1: on using this kind of globals, since we already keep track of the modules what if we do a second transverse at the end for every module, analyzing whether we should replace the callbacks or not?